spec/big_files/big_files_spec.rb

```diff
@@ -14,14 +14,18 @@
   end
 
   [{ glob: nil, exclude: nil },
-   { glob: '*/*.{rb,swift}', exclude: '*/foo.rb' }].each do |config|
+   { glob: '*/*.{rb,swift}', exclude: '*/foo.rb' },
+   { glob: '*/*.{rb,swift}', exclude: '*/foo.rb',
+     num_files: '6' }].each do |config|
     glob = config[:glob]
     exclude_glob = config[:exclude]
+    num_files = config[:num_files]
     context "With glob of #{glob}" do
       subject(:args) do
         args = []
         args += ['--glob', glob] unless glob.nil?
         args += ['--exclude', exclude_glob] unless exclude_glob.nil?
+        args += ['--num-files', num_files] unless num_files.nil?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.